parts/header.html

```diff
@@ -1,13 +1,22 @@
 <!-- wp:group {"align":"wide","style":{"spacing":{"padding":{"top":"20px","bottom":"20px"}}},"backgroundColor":"base","layout":{"type":"constrained"}} -->
-<div class="wp-block-group alignwide has-base-background-color has-background" style="padding-top:20px;padding-bottom:20px"><!-- wp:group {"align":"wide","layout":{"type":"flex","justifyContent":"space-between","flexWrap":"wrap"}} -->
-	<div class="wp-block-group alignwide"><!-- wp:group {"style":{"spacing":{"blockGap":"var:preset|spacing|20"},"layout":{"selfStretch":"fit","flexSize":null}},"layout":{"type":"flex"}} -->
-	<div class="wp-block-group"><!-- wp:site-logo {"width":60 } /-->
+<div class="wp-block-group alignwide has-base-background-color has-background"
+	style="padding-top:20px;padding-bottom:20px">
```

Comment:
Haven't tested this, but breaking actual HTML tag could be an issue here.

Reply:
yeah, I suspect this as well

Reply:
FYI, other than this I personally haven't experienced any issue with beautifying block HTML when building my own block theme.